src/layouts/Layout.astro

```diff
@@ -95,9 +95,7 @@ const footerIcons = [
         </div>
       </div>
     </nav>
-    <div>
-      <slot />
-    </div>
+    <slot />
```

Comment:
I think removing this `div` has some side effects with the `display: grid` of the body.

![image](https://github.com/amplifying-fsharp/amplifying-fsharp.github.io/assets/2621499/e5fa2d36-dc0c-4806-9d72-9fb3d2c1eb1d)

Reply:
Yeah looks like we can now just use the default `grid-template-rows` now with everything set to `auto` implicitly and that looks like it fixes it for me now